docs/reference/commandline/network_rm.md

```diff
@@ -10,18 +10,35 @@ parent = "smn_cli"
 
 # network rm
 
-    Usage:  docker network rm [OPTIONS] NAME | ID
+    Usage:  docker network rm [OPTIONS] NETWORK [NETWORK...]
 
-    Deletes a network
+    Deletes one or more networks
 
       --help=false       Print usage
 
-Removes a network by name or identifier. To remove a network, you must first disconnect any containers connected to it.
+Removes one or more networks by name or identifier. To remove the network, you must first disconnect any containers connected to it.
+
+To remove one network named 'my-network':
```

Comment:
oh, looks like you updated the wrong line; this line should be `To remove the network...`. The line above can still use "to remove a network"